src/scene/vcPolyModelNode.cpp

```diff
@@ -151,18 +151,22 @@ void vcPolyModelNode::HandleImGui(vcState *pProgramState, size_t *pItemID)
     m_matrix.extractTransforms(position, scale, orientation);
 
     bool repackMatrix = false;
-    if (ImGui::InputScalarN(vcString::Get("sceneModelPosition"), ImGuiDataType_Double, &position.x, 3))
+    ImGui::InputScalarN(vcString::Get("sceneModelPosition"), ImGuiDataType_Double, &position.x, 3);
+
+    if (ImGui::IsItemDeactivatedAfterEdit())
       repackMatrix = true;
 
     udDouble3 eulerRotation = UD_RAD2DEG(orientation.eulerAngles());
-    if (ImGui::InputScalarN(vcString::Get("sceneModelRotation"), ImGuiDataType_Double, &eulerRotation.x, 3))
+    ImGui::InputScalarN(vcString::Get("sceneModelRotation"), ImGuiDataType_Double, &eulerRotation.x, 3);
+
+    if (ImGui::IsItemDeactivatedAfterEdit())
     {
       repackMatrix = true;
       orientation = udDoubleQuat::create(UD_DEG2RAD(eulerRotation));
     }
 
-    if (ImGui::InputScalarN(vcString::Get("sceneModelScale"), ImGuiDataType_Double, &scale.x, 3))
-      repackMatrix = true;
+    ImGui::InputScalarN(vcString::Get("sceneModelScale"), ImGuiDataType_Double, &scale.x, 1);
```

Comment:
This doesn't have the deactivated code at all and poly models support 3 axis scaling

Reply:
Fixed